### PR TITLE
update TFMs to 2.0

### PR DESCRIPTION
**Pull request by mairaw**
@terrajobst can you take a look at this one? The .NET Standard version seemed to be wrong earlier.

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/standard/frameworks?branch=pr-en-us-2964)

### CODE REVIEW

**Comment by guardrex** [comment]
> The .NET Standard version seemed to be wrong earlier.

The last update was completed (merged) prior to the release of 2.0, so I couldn't say at the time "this is current" and list 2.0. https://github.com/dotnet/docs/pull/2592

**Comment by guardrex** [comment]
It's needs just a few more, too ... especially the lists ... see the ones I put in on https://github.com/dotnet/docs/pull/2950.

File: docs/standard/frameworks.md [hunk 1]
```diff
@@ -28,11 +25,11 @@ For package-based target frameworks (for example, .NET Standard and .NET Core),
...
 
-| Target Framework      | Latest Version | Target Framework Moniker (TFM) | .NET Standard Version | Metapackage |
-| :-------------------: | :------------: | :----------------------------: | :-------------------: | :---------: |
-| .NET Standard         | 2.0.0          | netstandard2.0                 | N/A                   | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library) |
-| .NET Core Application | 2.0.0          | netcoreapp2.0                  | 2.0                   | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
-| .NET Framework        | 4.7            | net47                          | 1.5                   | N/A |
+| Target Framework      | Latest Version | Target Framework Moniker (TFM)   | Minimum <br/> .NET Standard Version | Metapackage |
```
Comment (terrajobst) [inline docs/standard/frameworks.md]: We shouldn't say minimum. We should probably say "Implemented .NET Standard Version".

Logically, it's the maximum .NET Standard version that you can run on the given platform.

File: docs/standard/frameworks.md [hunk 2]
```diff
@@ -28,11 +25,11 @@ For package-based target frameworks (for example, .NET Standard and .NET Core),
...
+| :-------------------: | :------------: | :------------------------------: | :---------------------------: | :---------: |
+| .NET Standard         | 2.0.0            | netstandard2.0                 | N/A                           | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library) |
+| .NET Core Application | 2.0.0            | netcoreapp2.0                  | 2.0                           | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
+| .NET Framework        | 4.7            | net47                            | 1.0                           | N/A |
```
Comment (terrajobst) [inline docs/standard/frameworks.md]: .NET Framewor 4.7 also implements .NET Standard 2.0.

**Review by terrajobst — CHANGES_REQUESTED** [review]

File: docs/standard/frameworks.md (hunk 2 above)
Comment (mairaw) [reply docs/standard/frameworks.md]: What I'm trying to say is the lowest version of the standard that you can target. That's why I went with minimum. We could also remove that column too.

**Comment by mairaw** [comment]
Does this look right to you now @terrajobst? Otherwise, I might just remove that column.

**Comment by terrajobst** [comment]
I think the column is super valuable. But honestly, I think we should just say "Implemented .NET Standard Version" and remove the range. The lower bound is implied by regular rules. We don't say that .NET Framework 4.6.1 supports .NET Framework 1.0 - 4.6.1 either 😄

**Comment by mairaw** [comment]
Makes sense. See if that looks good now @terrajobst. Appreciate the review!

**Review by terrajobst — APPROVED** [review]

**Comment by terrajobst** [comment]
Looks good, but should we remove the last column? We've de-emphasized the packages and these are automatically referenced.

**Comment by mairaw** [comment]
Makes sense. I'll make the changes before merging. Thanks again!

**Pull request merged**